cmd/server.go

```diff
@@ -63,16 +63,14 @@ be displayed and one can be selected.
 	}
 }
 
-// serverCurrent command to display the name of the current server in the local config
-func serverCurrent(ctx *cli.Context) error {
-	cf, err := loadConfig(ctx)
+// printCurrentServer command to display the name of the current server in the local config
+func printCurrentServer(ctx *cli.Context) error {
+	serverName, url, err := getCurrentServer(ctx)
 	if err != nil {
 		return err
 	}
 
-	serverName := cf.CurrentServer
-	URL := cf.Servers[serverName].URL
-	fmt.Printf("Name: %s URL: %s\n", serverName, URL)
+	fmt.Printf("Name: %s URL: %s\n", serverName, url)
```

Comment:
What is the purpose of this change? It doesn't seem related to the rest of the PR

Reply:
I set this up prior to moving to the config approach. I wanted to put it in common to be accessed elsewhere. No longer needed, will remove.